cdap-ui/app/services/adapter/my-adapter-templatep-api.js

```diff
@@ -0,0 +1,21 @@
+angular.module(PKG.name + '.services')
+  .factory('myAdapterTemplatesApi', function($resource) {
+    return $resource(
+      '',
+      {
+        apptype: '@apptype',
+        appname: '@appname'
+      },
+      {
+        query: {
```

Comment:
They both ideally mean the same. Don't know which one is the correct usage. If query sounds not intuitive will change it though. @elfenheart thoughts?